src/js/sdk.ts

```diff
@@ -29,12 +29,7 @@ const DEFAULT_OPTIONS: ReactNativeOptions = {
 /**
  * Inits the SDK
  */
-export function init(
-  passedOptions: ReactNativeOptions = {
-    enableNative: true,
-    enableNativeCrashHandling: true,
```

Comment:
`m`: Why did we remove this?

Reply:
It was redundant given we merge it with `DEFAULT_OPTIONS` https://github.com/getsentry/sentry-react-native/blob/939e067be94cf3951855b05e3948a542c4b2e49e/src/js/sdk.ts#L22-L27